src/renderer/utils/createStorage.ts

```diff
@@ -73,11 +72,6 @@ export function createAppStorage<T>(key: string, defaultValue: T, clusterId?: st
       equals: comparer.structural, // save only when something really changed
     });
 
-    // remove json-file when cluster deleted
```

Comment:
Does it get removed some other way now?

Reply:
So I removed this for two reasons, first of all because the current logic is broken. No cluster will ever be part of `removedClusters` because of how sync works.

Second because it was a circular import. I think a better form would be a "force delete" option when removing a cluster.